Add Ledger Connection Troubleshoot section (#240)
crypto-org-chain/chain-docs: docs/wallets/ledger_desktop_wallet.md

# Using Ledger device with desktop wallet

### Crypto.org Chain Desktop Wallet (Beta)

> Crypto.org Chain Desktop Wallet (Beta) supports operations ranging from sending multiple assets, staking, governance & NFT. latest version can be found on the [release page](https://github.com/crypto-com/chain-desktop-wallet/releases). 

Reviewer: ```suggestion 
Crypto.org Chain Desktop Wallet (Beta) supports operations ranging from sending multiple assets, staking, governance & NFT. The latest version can be found on the [release page](https://github.com/crypto-com/chain-desktop-wallet/releases). 
```

## Restoring a Ledger wallet 

> Since the private key of Ledger wallets are always stored inside the Ledger device, you don't need the **Mnemonic Phrases** to restore a Ledger wallet on the Desktop Wallet. You may just follow our [*Create wallet in Desktop Wallet with Ledger device Guide*](#step-2-create-wallet-in-desktop-wallet-with-ledger-device) once again to restore your previously created Ledger wallet. 

Reviewer: ```suggestion
Since the private key of Ledger wallets is always stored inside the Ledger device, you don't need the **Mnemonic Phrases** to restore a Ledger wallet on the Desktop Wallet. You may also follow the [*Create wallet in Desktop Wallet with Ledger device Guide*](#step-2-create-wallet-in-desktop-wallet-with-ledger-device) to restore your previously created Ledger wallet. 
```

Please note that you may still need to input the correct **Address Index** to derive your desired Ledger wallet address.

## Ledger connection troubleshoot

> There are numerous reasons & possibilities why your Ledger device cannot be connected with the Crypto.org Chain Desktop Wallet (Beta) properly. Here are the most common checklist for you to troubleshoot the issue: 

Reviewer: ```suggestion
Most ledger users are able to connect to Crypto.org Chain Desktop Wallet (Beta) efficiently, while some users may encounter connection issues. Below are the most common reasons that cause the connection problem between a Ledger device and the Crypto.org Chain Desktop Wallet (Beta) and a checklist that you may refer to troubleshoot the issue: 
```

Author: That's a much better rephrasing~ haha

- Use the latest version of "Crypto.org App" & "Ethereum App" on Ledger
- Wait several seconds after the App on Ledger device is first opened
- If you have connected the Ledger device on other applications, e.g. Ledger Live, close these applications since they could potentially occupy the Ledger connection channel

> - Restart your machine to clear off any potential connection channel occupency

Reviewer: ```suggestion
- Restart your machine to clear off any potential connection channel occupancy
```
